modules/graphstudio/pages/map-data-to-graph.adoc

```diff
@@ -145,20 +132,43 @@ The folder preview, like the file preview, is limited to the first ten lines of
 If a folder contains more than one file and the first file has more than ten lines, only the first ten lines of the first file will appear in the preview.
 
 [NOTE]
-====
 TigerGraph supports loading from archived and compressed GCS files directly.
 Currently supported file extensions include zip, tar.gz, tgz and tar.
 GraphStudio detects the file extension and automatically chooses the corresponding file format.
 If the file is encoded with one of these formats but has a non-standard file extension, you can manually specify the file format.
-====
 
 Confirm that the data has parsed correctly (see <<Configure the File Parser>>) and click btn:[ADD] to add the data source to your project, where it will appear in the working panel.
 
-Continue with <<Map data files to vertex type or edge type>>.
+--
+Azure Blob Storage::
++
+--
```

Comment:
I think we are missing the step where users have to input credentials for Azure blob storage here